src/query.rs

```diff
@@ -52,4 +52,46 @@ impl Query {
             inner: Box::new(inner),
         })
     }
+
+    /// Construct a Tantivy's FuzzyTermQuery
+    #[staticmethod]
+    #[pyo3(signature = (schema, field_name, text, distance = 1, transposition_cost_one = true))]
```

Comment:
The default values are my preference, but I have no very strong opinions on them.